pkg/parts/parts.go

```diff
@@ -109,9 +167,39 @@ type Part struct {
 	Sheets Links `json:"sheet,omitempty"`
 }
 
+func (x *Part) RedisKey() string {
+	return fmt.Sprintf("%s:%s:%d", x.ID.Project, x.ID.Name, x.ID.Number)
+}
+
+func (x *Part) DecodeRedisKey(str string) {
+	got := strings.SplitN(str, ":", 3)
+	x.ID.Project = got[0]
+	x.ID.Name = got[1]
+	num, _ := strconv.Atoi(got[2])
+	x.ID.Number = uint8(num)
+}
+
+func (x *Part) ZScore() int {
+	projectScore, _ := strconv.Atoi(x.ID.Project[:2])
+	return projectScore
+}
```

Comment:
Maybe we should split off numerical ID from string token.

Reply:
Nah. I mean... we aren't even using the zscore yet. Not even sure I should be scoring this at all.

Reply:
Fari enough.